Dockerfile

```
@@ -38,8 +38,8 @@ COPY .babelrc \
     favicon.png \
     /app/
 
-ARG NODE_ENV=${NODE_ENV}
-RUN npm run build:${NODE_ENV}
+ARG DEPLOY_ENV=${DEPLOY_ENV}
+RUN npm run build:${DEPLOY_ENV}
```

We should get rid of the different builds here, instead the Docker image should always build a production ready build.

```suggestion
RUN npm run build:production
```